p/i/index.php

```diff
@@ -76,11 +76,8 @@
 	}
 
 	if ($error) {
-		// TODO this should be definitely improved to display a nicer error
-		// page to the users (especially non administrators).
-		echo '### Fatal error! ###<br />', "\n";
 		Minz_Log::error($error);
-		echo 'See logs files.';
+		errorMessage ('Fatal error');
```

Comment:
```suggestion
		errorMessage('Fatal error');
```

Reply:
https://github.com/FreshRSS/FreshRSS/blob/master/lib/Minz/FrontController.php#L70
I did this because it looked like the used code style in the files (this example is even committed by you), should I then change all functions that do this?

Reply:
Yes please, no space between parenthesis in our current convention